processing/thermal.py

```diff
@@ -60,18 +60,51 @@ def process(recording, conf):
             classify(conf, recording, api, s3, logger)
 
 
-def classify(conf, recording, api, s3, logger):
-    working_dir = recording["filename"].parent
-    command = conf.classify_cmd.format(
-        folder=str(working_dir), source=recording["filename"].name
-    )
+def classify_models(api, command, conf):
+    """ classifies all models described in the config """
 
-    logger.debug("processing %s", recording["filename"])
+    model_results = []
+    main_model = None
+
+    for model in conf.models:
+        model_result = classify_file(api, command, conf, model=model)
```

Comment:
If we run classify_file twice aren't we finding the tracks twice and generating the MP4 twice?   Would it not be better to change the pipeline so that it could run the tracks/mp4s once and then classify using two pipelines?   (Although this may result in more changes and not be worth it I think we should consider it as this code will live for a long time)